### PR TITLE
Added jss-vendor-prefixer

### PR DESCRIPTION
**Pull request by HenriBeck**
This fixes the tests🙌

### CODE REVIEW

**Comment by kof** [comment]
jss-plugin-vendor-prefixer?

**Comment by HenriBeck** [comment]
yeah, I waited for the name change because I wasn't quite sure about this one.

File: packages/jss-vendor-prefixer/src/index.test.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+    beforeEach(() => {
+      sheet = jss.createStyleSheet({
+        a: {animation: 'yyy'}
+      })
+    })
+
+    it('should generate correct CSS', () => {
+      if (isIEorEdge) {
+        expect(sheet.toString()).to.be('@keyframes a {\n}')
```
Comment (HenriBeck) [inline packages/jss-vendor-prefixer/src/index.test.js]: Why will this be `keyframes` @kof? Doesn't make sense to me. This test fails in the BrowserStack tests but runs successfully locally.

File: packages/jss-vendor-prefixer/src/index.test.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  describe('@keyframes', () => {
+    let sheet
+
+    beforeEach(() => {
+      sheet = jss.createStyleSheet({
+        '@keyframes a': {}
+      })
+    })
+
+    it('should generate correct CSS', () => {
```
Comment (HenriBeck) [inline packages/jss-vendor-prefixer/src/index.test.js]: Also, this one fails in the BrowserStack this test fails as well.

File: packages/jss-vendor-prefixer/src/index.test.js (hunk 1 above)
Comment (kof) [reply packages/jss-vendor-prefixer/src/index.test.js]: doesn't make sense to me, cc @AleshaOleg https://github.com/cssinjs/jss-vendor-prefixer/commit/739a34128c2d84395fc441093710bf1cedd4a9b8

File: packages/jss-vendor-prefixer/src/index.test.js (hunk 2 above)
Comment (HenriBeck) [reply packages/jss-vendor-prefixer/src/index.test.js]: Apparently, the keyframes aren't being prefixed in IE 9.0.0 (Windows 7.0.0), IE 10.0.0 (Windows 8.0.0), IE 11.0.0 (Windows 8.1.0.0), Edge 15.15063.0 (Windows 10.0.0)

Comment (HenriBeck) [reply packages/jss-vendor-prefixer/src/index.test.js]: So they assertion is wrong.

Comment (kof) [reply packages/jss-vendor-prefixer/src/index.test.js]: the assertion is not running, see early exit before on ln 24

Comment (kof) [reply packages/jss-vendor-prefixer/src/index.test.js]: wait, early exit is only IE9, others should not add any prefixes afaik, cc @AleshaOleg how did the tests even pass?

Comment (HenriBeck) [reply packages/jss-vendor-prefixer/src/index.test.js]: That's a different test. Two tests are failing right now

Comment (HenriBeck) [reply packages/jss-vendor-prefixer/src/index.test.js]: I think the jss-vendor-prefixer plugin wasn't tested with BrowserStack and locally those tests succeed.

Comment (HenriBeck) [reply packages/jss-vendor-prefixer/src/index.test.js]: Ohhh I know whats going on here...

The check for IE and Edge is supposed to be in this test and not in the other test.

That way the tests make sense...

Comment (HenriBeck) [reply packages/jss-vendor-prefixer/src/index.test.js]: @kof tests seem to pass now

**Comment by kof** [comment]
already approved

**Pull request merged**